python/ray/tests/test_task_events.py

```diff
@@ -286,50 +290,6 @@ def f():
     )
 
 
-def test_fault_tolerance_parent_failed(shutdown_only):
```

Comment:
this test is now better covered in test_task_events_2.py:: test_fault_tolerance_chained_task_fail